jsprit-core/src/main/java/com/graphhopper/jsprit/core/problem/io/VrpXMLWriter.java

```diff
@@ -67,6 +69,14 @@ public Document createDoc() throws ConfigurationException {
 
     private Collection<VehicleRoutingProblemSolution> solutions;
 
+    private Boolean only_best_solution = false;
```

Comment:
Thanks. I d prefer the primitive type boolean over the object here, and, would you mind to change the name acc to lower camelCase? (see conventions here for example: http://www.oracle.com/technetwork/java/codeconventions-135099.html).